### ISSUE TITLE
Offering Sign_Lines to a sign doesn't send a update packet

### ISSUE THREAD

**Issue by clienthax** [SpongePowered/SpongeCommon#239]
``` java
Text[] texts = {Texts.of("1"), Texts.of("2") };
ArrayList<Text> list = Lists.newArrayList(texts);
getLocation().offer(Keys.SIGN_LINES, list);
```

Where getLocation is a Location<World> of a placed sign
will not update the text on the client view until the player relogs

**Comment by Kiskae** [comment]
Looking at the data processor (for SignData):
https://github.com/SpongePowered/SpongeCommon/blob/master/src/main/java/org/spongepowered/common/data/processor/data/tileentity/SignDataProcessor.java#L163
and the value processor (for Keys.SIGN_LINES):
https://github.com/SpongePowered/SpongeCommon/blob/master/src/main/java/org/spongepowered/common/data/processor/value/tileentity/SignLinesValueProcessor.java#L64
Seems like this can be fixed by marking the location in the world for a reset like in the data processor

**Comment by clienthax** [comment]
https://github.com/SpongePowered/SpongeCommon/pull/240

**Comment by gabizou** [comment]
Fixed with the closure of #240 .
